### DISCUSSION TITLE
[建议]我也搭建了一套，请问如何实现英文

### DISCUSSION THREAD

**Discussion by geeeeeeeek** [win12-online/win12#293] [Q&A] [answered]
### 预览地址

http://win12.gitapp.cn

请问， 如何实现英文布局呢

**Comment by NB-Group** [comment, marked as answer]
我们正在进行翻译工作，但是工作量实在太大，请耐心等待

**Comment by ghost** [comment]
gitapp是什么鬼

**Comment by NebulaTechs** [comment]
不出意外的话，在年底之前可以完成

**Comment by ghost** [comment]
年底。。。

**Comment by User782Tec** [comment]
> 年底。。。

我们最近学业还是比较繁重的(

**Comment by tjy-gitnub** [comment]
不出意外的话我们是完不成翻译工作的（

**Reply by NB-Group** [reply]
哦哦哦！！！谭景元！！！

**Reply by tjy-gitnub** [reply]
6

**Comment by lingbopro** [comment]
话说你们是怎么实现多语言切换的呢

**Reply by tjy-gitnub** [reply]
实现了吗（大雾

**Reply by lingbopro** [reply]
要翻译多语言的话肯定得有多语言切换啊

**Reply by tjy-gitnub** [reply]
现在实现了（、、两年过去了（bs

**Reply by lingbopro** [reply]
> 现在实现了（、、两年过去了（bs

啊？

**Reply by tjy-gitnub** [reply]
....翻译工作 (2023/7~2025/1)

**Reply by NB-Group** [reply]
艰辛（）

**Comment by Mbilse** [comment]
> ### 预览地址
> 
> http://win12.gitapp.cn
> 
> 请问， 如何实现英文布局呢
> 
这不就是我们谈论的冒牌货吗？
问一下只是你纯手工打造的吗

**Comment by lingbopro** [comment]
@geeeeeeeek 原来是你搭的冒牌货……

那我问你，为什么不注明是修改版

**Comment by tjy-gitnub** [comment]
原来是你 ^u^)o @geeeeeeeek
广告费分一点呗~

**Reply by kjmjh** [reply]
()
